### DISCUSSION TITLE
User-triggered tuning from AF- and EON-list

### DISCUSSION THREAD

**Discussion by lxsxl** [PE5PVB/TEF6686_ESP32#425] [Ideas]
You could make the frequencies in AF- and EON-list selectable by the rotation-knob (similar to menu-items), and with pressing the knob switch to the selected frequency. The view should remain in the active screen (AF or EON). With switching to a new frequency of course clear all RDS codes. It can be discussed, which frequencies are useful for this function (imho at least the green AFs and the EON-frequencies, maybe also the red AFs (as they carry a receivable station, the yellow ones should be the least important).

**Comment by PE5PVB** [comment]
For AF this shouldn't be a problem, for EON it's difficult, because RDS data will be overwritten.

**Comment by lxsxl** [comment]
On frequency switch there should be a complete clear of all data, no matter if it is an AF or an ON. Or am I wrong?

**Comment by PE5PVB** [comment]
Yes, when switching to another station, but when swithing to another AF, you should not reset it.

**Comment by lxsxl** [comment]
But isn‘t that the actual behavior? I don‘t have the TEF with me but til now I was sure that it would clear also if it tunes to an AF.

**Comment by lxsxl** [comment]
I just checked it: Even on AF-tuning to the same PI-code there is a complete clear of all RDS data.
